Ccs.Ppg.NotificationService.API/Controllers/NotificationController.cs

```diff
@@ -45,5 +47,13 @@ public async Task<bool> Post(MessageRequestModel message)
         {
             return await _messageProviderService.SendMessage(message);
         }
-    }
-}
+
```

Comment:
add description for this API, take reference of sms API

Reply:
Done